Document user questions (docs.daml.com) (#327)
digital-asset/docs.daml.com: docs/2.7.0/docs/daml-finance/tutorials/getting-started/lifecycling.rst

* The party executing settlement can be chosen as well, as described in the previous tutorial on
  :doc:`Settlement <settlement>`.

Which party should take the role as *lifecycler*?
=================================================

> From a design perspective, a lifecycler should be the party that defines the lifecycle events

Reviewer: The event can be defined by parties other than the life-cycler.

The life-cycler is currently trusted with
- timely and complete event processing
- providing accurate observations

Author: I added both points to the answer now.

happening on an instrument. In the simplified example above, it is the bank. In a more realistic
example, it would probably be the issuer.
In some special cases like Elections, if we really need the owner to be the lifecycler in order to
choose which "path" they'd like to lifecycle the instrument with, we can use a delegation contract.

Which party is the provider of the Effect?
==========================================

Most of the time the provider of the Effect is the lifecycler, yet in some cases where we want to

> hide the visibility of holdings and the calculation result, we will have a provider that is

Author: > I don't understand the part on the calculation result.

I think the original answer in the Issue refers to the consumed/produced instruments.

> If we don't want the lifecycler to see this, then we need a different effect provider.

I think that is what the original answer says? However, maybe in a more complicated way. I have changed the wording now to what you wrote.

* The party executing settlement can be chosen as well, as described in the previous tutorial on
  :doc:`Settlement <settlement>`.

Which party should take the role as *lifecycler*?
=================================================

From a design perspective, a lifecycler is often the party that defines the lifecycle events
happening on an instrument (although they can be different). In the simplified example above, it is
the bank. In a more realistic example, it would probably be the issuer.
In some special cases like Elections, if we really need the owner to be the lifecycler in order to

> choose which "path" they'd like to lifecycle the instrument with, we can use a delegation contract.

Reviewer: ```suggestion
choose which "path" they would like to lifecycle the instrument with, we can use a delegation contract.
```

Reviewer: I would remove the part on elections as it' quite out of context. At this stage the reader hasn't even seen an example with elections

Author: I took out that part now, and kept the part with "some special cases ... delegation contract"

The lifecycler is currently trusted with:

* Timely and complete Event processing
* Providing accurate Observations

Which party is the provider of the Effect?
==========================================

Most of the time the provider of the Effect is the lifecycler. However, in some cases we may want to

> hide the visibility of holdings and the calculation result. The provider of the Effect gets to see

Reviewer: ```suggestion
hide the visibility of claimed holdings. The provider of the Effect gets to see
```

> all holdings claimed against that one Effect contract. If we don't want the lifecycler to see this,

Reviewer: ```suggestion
all holdings claimed against that one Effect contract. If we wish to avoid that,
```

> then we need a different effect provider.

Reviewer: ```suggestion
we then need a different effect provider.
```

  now simply add a distribution rule to the running system to enable this new lifecycle event for
  their instrument without affecting the actual live instrument itself (or any holdings on it).

> Can I integrate a holding ownership change within lifecycling?

Author: I have now clarified that the question concerns the target instrument. Also, added your dividend example.
